driver/lib/commands/context.ts

```diff
@@ -1,13 +1,27 @@
 import { FlutterDriver } from '../driver';
 
 export const FLUTTER_CONTEXT_NAME = `FLUTTER`;
+export const NATIVE_CONTEXT_NAME = `NATIVE_APP`;
 
 export const getCurrentContext = function(this: FlutterDriver) {
   return this.currentContext;
 };
 
-export const setContext = function(this: FlutterDriver, context: string) {
-  return (this.currentContext = context);
+export const setContext = async function(this: FlutterDriver, context: string) {
+  if ([FLUTTER_CONTEXT_NAME, NATIVE_CONTEXT_NAME].includes(context)) {
+    this.proxyWebViewActive = false;
+    // Set 'native context' when flutter driver sets the context to FLUTTER_CONTEXT_NAME
+    if (this.proxydriver) {
+      await this.proxydriver.setContext(NATIVE_CONTEXT_NAME);
```

Comment:
What if user wants go back to Flutter context from Native?

Reply:
Flutter driver handles its context with `this.currentContext`.
`this.proxydriver.setContext` is to handle context in UIA2/XCUITest drivers, so this means, UIA2/XCUITest driver set native context in both FLUTTER_CONTEXT_NAME and NATIVE_CONTEXT_NAME cases.